frontend/hub/collections/CollectionDetails.tsx

```diff
@@ -1,655 +0,0 @@
-import {
```

Comment:
@MilanPospisil What is the reason for deleting this file? I see it referenced in https://github.com/ansible/ansible-ui/pull/682/files#diff-2c6090bf33d9586c01409f00fa9b0c30951fc7a24ef727b8ff43259cc9171a3cR23

Reply:
This PR will be from scratch anyway, Shaiah made some collection list in namespace detail, so I rather close this.